test/screentest/runner.js

```diff
@@ -22,6 +22,9 @@ const args = process.argv.slice(2);
 
 (async function main() {
 
+  console.log('pause');
+  await new Promise(r => setTimeout(r));
```

Comment:
Can you add a comment on why this defer is needed?

Reply:
it's here from another stashed change. Will remove it